include/gatekeeper_lls.h

```diff
@@ -211,6 +211,16 @@ struct lls_config {
 };
 
 /*
+ * TODO Enable an lcore to subscribe multiple times for the same IP address.
+ * This is needed because it's very hard for the GK blocks avoid duplicates.
+ * Currently, the LLS block has trouble with unsubscribing those duplicates
+ * since the LLS block identifies what to remove using
+ * the pair <IP_address, lcore_id>.
+ * In addition, the LLS block might not accept a subscription because
+ * it saves memory for a single subscription for lcore id.
+ */
+
+/*
```

Comment:
I think the GK blocks may subscribe the same IP addresses by an lcore for multiple times, since different IP prefixes may have a reference to the same FIB entry, it's possible to subscribe multiple times for the same IP address. However, the GK blocks will only call ``put_xxx()`` to unsubscribe the entry only when no FIB entry points to it. So, I think the GK blocks have already handled this issue. There should be no bugs for this patch. What do you think?